src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -320,33 +320,81 @@ bool LottieLoader::frame(float no)
     no = roundf(no);
     no *= 0.001f;
 
-    this->frameNo = no;
+    this->frameNo = no + startFrame();
 
     TaskScheduler::request(this);
 
     return true;
 }
 
```

Comment:
please make it sure the code-consistency that inserting 2 spaces between 2 methods/functions. (other file as well)